### PR TITLE
(#1670, #1663, #1623, #1624, #1706) Media meta description rename; update meta description max length; constrain body fields of block content types; rearrage Video fields

### LINKED ISSUE #1623: The body field of the raw_html_block block_content entity should be constrained to raw_html text format
The body field raw_html_block content_block entity should be constrained to raw_html text format.

Additionally any existing YAML needs to be updated to use the new format. This is related to #1192

### LINKED ISSUE #1624: The body field of the content_block block_content entity should be constrained to full_html text format
The body field of the content_block block_content entity should be constrained to full_html text format.

Any existing YAML should be updated. There should not be any that I can think of. This is related to #1192.

### LINKED ISSUE #1663: Update Meta Description Field to allow 600 characters
Because PDQ has meta descriptions longer than 320 characters, we need to update the meta description field to allow 600 characters.

This is needed to continue migration scripting.

### LINKED ISSUE #1670: Video and Infographic — Change field_meta_description to field_page_description
Our content types use the backend name of `field_page_description` for the meta description field. It seems a few media types missed this memo.

This ticket is to make **Video** and **Infographic** consistent with the other content types, changing `field_meta_description` to `field_page_description`. 

Note: this is **only** a change to the backend field name to match the [requirements](https://github.com/NCIOCPL/cgov-digital-platform/wiki/Common-Fields#Meta-Description). The label should still be Meta Description.

### PR DESCRIPTION
**Pull request by sarinapadilla**
Closes #1670
- [x] Rename field_meta_description to field_page_description
- [ ] Update field and form/view displays for video and infographic
- [ ] Update video and infographic YAML content

Closes #1663
- [ ] Allow 600 chars in content and media meta descriptions
  - [x] Media
  - [ ] Content

Closes #1623
- [ ] Create new field_raw_html for block_content types
- [ ] Add field definition for field_raw_html on raw_html_block type
- [ ] Add field_raw_html to form/view displays for raw_html_block
- [ ] Update getMegaMenuContent to check for field_raw_html
- [ ] Update NavItem to check for field_raw_html
- [ ] Update migration file with correct field mapping
- [ ] Update bio/press release Raw HTML Blocks created on install
- [ ] Update Raw HTML Block YAML content

Closes #1624
- [ ] Constrain cgov_content_block body fields to full_html text format

Closes NCIOCPL#1706
- [ ] Move YouTube URL field above Site Section
- [ ] Temporary fix for issue with setting YouTube URL after Site Section

### CODE REVIEW

**Comment by sarinapadilla** [comment]
ODE for testing: http://ncigovcdode132.prod.acquia-sites.com/

File: docroot/profiles/custom/cgov_site/modules/custom/cgov_core/src/NavItem.php [hunk 1]
```diff
@@ -226,7 +226,12 @@ public function getMegamenuContent() {
     $referencedEntities = $megamenuFieldEntityReference->referencedEntities();
     $hasMegamenu = count($referencedEntities) > 0;
     if ($hasMegamenu) {
-      $megamenuMarkupEncoded = $megamenuFieldEntityReference->entity->get('body')->value;
+      if ($megamenuFieldEntityReference->entity->hasField('body')) {
+        $megamenuMarkupEncoded = $megamenuFieldEntityReference->entity->get('body')->value;
+      }
+      elseif ($megamenuFieldEntityReference->entity->hasField('field_raw_html')) {
```
Comment (bryanpizzillo) [inline docroot/profiles/custom/cgov_site/modules/custom/cgov_core/src/NavItem.php]: Just use field_raw_html for this, and then constrain the Site Section to only allow raw html. That will be more straightforward -- we are not going to have content blocks.

File: docroot/profiles/custom/cgov_site/modules/custom/cgov_yaml_content/content/000_nav_blocks.content.yml [hunk 2]
```diff
@@ -7,11 +7,11 @@
   status: 1
```
Comment (bryanpizzillo) [inline docroot/profiles/custom/cgov_site/modules/custom/cgov_yaml_content/content/000_nav_blocks.content.yml]: This content should have been deleted. I wonder how it is appearing here...
Yeah, it was removed in https://github.com/NCIOCPL/cgov-digital-platform/pull/1646.
And somehow it was added back in as part of https://github.com/NCIOCPL/cgov-digital-platform/commit/5ab0625c373eb98b629f516956b5ea939613bfb1#diff-509092f44cecf9a817d95f13e6ddae83

So, please kill this file.

**Comment by sarinapadilla** [comment]
@bryanpizzillo all updates that you suggested are now included.

**Pull request closed without merging**